### PR TITLE
test-vector: verify a request from the JS SDK (old signature format)

### PR DESCRIPTION
**Pull request by naure**
See https://github.com/Cerebellum-Network/ddc-schemas/pull/19/files

### CODE REVIEW

File: model/domain/signedpiece_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+type StoreRequest struct {
+	Body string
+	Cid  string
+}
+
+func TestStoreRequest(t *testing.T) {
+	raw, err := os.ReadFile(TEST_VECTOR)
+	if err != nil {
+		t.Skip("WARNING: no test vectors. Run: git submodule update --init")
```
[the remark this answers is not in the document]
Comment (svalasovich) [reply model/domain/signedpiece_test.go:29]: https://github.com/Cerebellum-Network/cere-ddc-sdk-js/blob/proto-module/.github/workflows/ci.yaml#L21

[the remark this answers is not in the document]
Comment (naure) [reply model/domain/signedpiece_test.go:29]: Great

[the remark this answers is not in the document]
Comment (naure) [reply model/domain/signedpiece_test.go]: There’s also another way:
image

[the remark this answers is not in the document]
Comment (svalasovich) [reply model/domain/signedpiece_test.go]: Thanks

**Pull request merged**